frontend/src/libs/components/input/input.tsx

```diff
@@ -56,8 +62,9 @@ const Input = <T extends FormFieldValues>({
           type={type}
           placeholder={placeholder}
           maxLength={maxLength}
+          autoComplete={autoComplete}
           className={getValidClassNames(
-            styles['input'],
+            styles[isPrimaryStyle ? 'input' : style],
```

Comment:
we need to pass here input as basic styles and override additional styles for chat-input or primary input
```suggestion
           styles['input'],
           isChatInput && styles['input-chat'],
```
For the same for textarea